pkg/template/worker_template.go

```diff
@@ -365,6 +365,12 @@ storage:
       contents:
         source: "data:text/plain;charset=utf-8;base64,{{  index .Files "conf/docker-daemon.json" }}"
 
+    - path: /root/.docker/config.json
+      filesystem: root
+      mode: 0644
+      contents:
+        source: "data:text/plain;charset=utf-8;base64,{{  index .Files "conf/kubelet-docker-config.json" }}"
+
```

Comment:
probabbly stupid question, isnt docker running as docker user? so why it would take creds from root?

Reply:
kubelet is using `/root/.docker/config.json`

Reply:
There is no even sunch user as docker

Reply:
ah right, it's kubelet pulling the images, not docker, sorry ;)